app/views/posts/show.html.erb

```diff
@@ -1 +1,16 @@
-<h1> Post Details </h1>
\ No newline at end of file
+<h1>Single post to a given user</h1>
+
+<section>
+      <h2>
+      Post #<%= @post.id %> by <%= @user.name %>
+      </h2>
+        <span>Comments: <%= @post.comments_counter %></span>
+        <span>, Likes: <%= @post.likes_counter %></span>
+    
+      <p> <%= @post.text %></p>
+    
+    <% @post.most_recent_comments.each_with_index do |comment| %>
+      <p><%= "#{comment.author.name}: #{comment.text}" %></p>
+    <% end %>

```

Comment:
- [ ] Let's follow the design from the given wireframes here as well. The comments are in a linear display instead of being displaying them line by line. Below is an image for your reference.

![comment](https://user-images.githubusercontent.com/53124548/230361742-3bbc8d1b-ca44-49ae-8a17-c9610dd9650c.png)

Reply:
Noted, I will fix this